Makefile

```diff
@@ -118,6 +118,7 @@ new_env: clean
 		pipenv install --dev --skip-lock;\
 		pipenv run pip install -e .[all];\
 		pipenv run pip install --no-deps file:plugins/aea-ledger-ethereum;\
+		pipenv run pip install --no-deps file:plugins/aea-ledger-ethereum-flashbots;\
```

Comment:
I think this would cause problems for py < 3.9, can we make this optional somehow and print out a warning for cases when it cannot be installed?

Reply:
I think you are right.